app/views/transactions.html

```diff
@@ -88,27 +98,34 @@ <h3 style="margin: 20px;" ng-if="transactionsSelected.length < defaultBulkRerunL
                 </tr>
               </thead>
               <tbody>
-                <tr class="table-list" ng-repeat="transaction in transactions" ng-click='viewTransactionDetails("transactions/" + transaction._id, $event)'>
+                <tr class="table-list" ng-repeat="transaction in transactions" ng-style="baseIndex + $index < 0 && {'color': '#036633', 'font-weight': 'bold'}" ng-click='viewTransactionDetails("transactions/" + transaction._id, $event)'>
```

Comment:
Sure, I'll try switching to a class.

So basically `baseIndex` decrements every time a new transaction comes in, helping to keep to track of which ones were loaded when the page was loaded. If a transaction has `baseIndex+$index` equal or greater than zero, we know it's one of the old transactions; below zero it's new. The old transactions can then keep their original index number in the display. This little ng-style bit applies styling to new transactions that come in.